app/views/legislation/questions/show.html.erb

```diff
@@ -29,6 +29,7 @@
   <div class="row">
     <div class="small-12 medium-9 column">
       <h3 class="quiz-question"><%= @question.title %></h3>
+      <%= sanitize(@question.description) %>
```

Comment:
Since this is a text edited in a `html-area admin` textarea:

```suggestion
      <%= AdminWYSIWYGSanitizer.new.sanitize(@question.description) %>
```